ci-operator/step-registry/baremetalds/packet/setup/OWNERS

```diff
@@ -1 +0,0 @@
-../OWNERS
```

Comment:
I wonder if you should add the maintainers one level up? 
i.e. in ci-operator/step-registry/baremetalds/packet

Reply:
it makes sense, I'll update